api/src/test/java/com/nicknathanjustin/streamercontracts/votes/VoteServiceImplTest.java

```diff
@@ -49,71 +50,97 @@ public void recordVote_nullContractModel_throwsException() {
 
     @Test(expected = NullPointerException.class)
     public void isVotingComplete_nullContractModel_throwsException() {
-        voteServiceImpl.isVotingComplete(null);
+        Optional<VoteModel> optionalProposerVote = Optional.empty();
+        Optional<VoteModel> optionalStreamerVote = Optional.empty();
+        voteServiceImpl.isVotingComplete(optionalProposerVote, optionalStreamerVote, null);
     }
 
     @Test(expected = NullPointerException.class)
     public void getVoteOutcome_nullContractModel_throwsException() {
         voteServiceImpl.getVoteOutcome(null);
     }
 
-    @Test(expected = IllegalArgumentException.class)
-    public void recordVote_contractIsCommunityContract_throwsException() {
-        voteServiceImpl.recordVote(createUserModel(PROPOSER_ID),
-                createContractModel(
-                        getValidContractModelBuilder().isCommunityContract(true),
+    @Test
+    public void recordVote_contractNotAccepted_returnsFalse() {
```

Comment:
test is misleading. should read:
```
recordVote_contractIsOpen_returnsFalse
```

Reply:
This throws an exception now but otherwise the test name has been updated appropriately.